hw5/nodes.py

```diff
@@ -0,0 +1,151 @@
+from dataclasses import dataclass
+from typing import Tuple, cast, Iterator, TypeVar, Generic
+
+from hw5.comparable import Comparable
+
+K = TypeVar("K", bound=Comparable)
+V = TypeVar("V", bound=Comparable)
+
+Entry = Tuple[K, V]
+
+
+class Node(Generic[K, V]):
```

Comment:
Sorry for my nerdy behaviour, but I would like to see docstrings in the code at least for key structures - Node, EmptyNode, NotEmptyNode, Treap